go.mod

```diff
@@ -4,7 +4,6 @@ go 1.13
 
 require (
 	github.com/blang/semver v3.5.1+incompatible
-	github.com/coreos/prometheus-operator v0.38.1-0.20200424145508-7e176fda06cc
```

Comment:
I feel like I remember this dependency being the one that forced us to use `replace` for `k8s.io/client-go`. Can you see if we can directly require `k8s.io/client-go v0.18.2` now?